README.md

```diff
@@ -362,7 +362,11 @@ Playbook:
                     1002
                 ],
                 "earliest_offset": 0,
-                "latest_offset": 42
+                "latest_offset": 42,
+                "at-min-isr": true,
```

Comment:
do you think we should keep the key in snake case format, just like `earliest_offset` and `latest_offset`?

Reply:
Done